site/docs/components/stepper-input/examples.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="stepper-input" exampleName="Default" >
+
+## Default
+
+Stepper input has a default value of 0, an increment value of 1 and a block value of 10, and you can only use it to accept numeric characters. A single click increases or decreases the value in the input by 1.
```

Comment:
Can't really test atm but does it actually have a default of 0?

Reply:
No, removed line